src/microbe_stage/systems/MicrobeMovementSystem.cs

```diff
@@ -194,6 +206,16 @@ private static float CalculateRotationSpeed(in Entity entity, ref OrganelleConta
             force *= cellProperties.MembraneType.MovementFactor -
                 (cellProperties.MembraneRigidity * Constants.MEMBRANE_RIGIDITY_BASE_MOBILITY_MODIFIER);
 
+            if (control.Sprinting && canSprint)
+            {
+                force *= Constants.SPRINTING_FORCE_MULTIPLIER;
+                strain.IsUnderStrain = true;
+            }
+            else
+            {
+                strain.IsUnderStrain = false;
```

Comment:
Shouldn't strain be unset when no movement keys are set? In the if checking `control.MovementDirection == Vector3.Zero`.